lib/FieldType/Mapper/AuthorFormMapper.php

```diff
@@ -58,15 +58,16 @@ public function mapFieldValueForm(FormInterface $fieldForm, FieldData $data)
         $fieldSettings = $fieldDefinition->getFieldSettings();
         $formConfig = $fieldForm->getConfig();
         $names = $fieldDefinition->getNames();
-        $label = $fieldDefinition->getName($formConfig->getOption('mainLanguageCode')) ?: reset($names);
+        $label = $fieldDefinition->getName($formConfig->getOption('languageCode'))
```

Comment:
Can we avoid duplication of this logic across `FieldValueMappers` implementations ?

Reply:
Was think about it, but either injecting service just for that or using trait seems like worse solutions than this. Unless you think different / have better option.